certbot/certbot/_internal/lock.py

```diff
@@ -228,8 +228,9 @@ def release(self):
             # https://github.com/python/typeshed/pull/3607 and included in
             # newer versions of mypy so it can be removed when mypy is
             # upgraded.
-            msvcrt.locking(self._fd, msvcrt.LK_UNLCK, 1)  # type: ignore
-            os.close(self._fd)
+            if self._fd:
```

Comment:
nit: Can we instead blow up in the case someone calls `release` when they haven't acquired the lock?

Reply:
Indeed, it makes more sense than silently do nothing.